collector/go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/pebbe/zmq4 v1.2.7
 	github.com/spf13/viper v1.10.1
 	github.com/stretchr/testify v1.7.0
+	github.com/sugary199/collector-version v0.0.0-20220323082224-e39a32dfb492
```

Comment:
I have checked this repository and found it is identical to `core.go`. This could be removed as you have introduced `core.go` to our base.